src/api/CustomerApi.ts

```diff
@@ -31,6 +34,17 @@ export const addNewCustomer = (customer: Customer) => {
   return customersSetter(customers);
 };
 
+
+export const deleteCustomer = (customerId: Key) => {
+  const customers = customersGetter();
+  const index = customers.findIndex((customer) => customer.id === customerId);
+
+  if (index !== -1) {
+    customers.splice(index, 1);
+    customersSetter(customers);
+  }
+};
+
```

Comment:
This looks like like the correct direction for the customer delete api.   I'd continue with and work on the component for the kabab and delete action